docs/azure_arc_jumpstart/azure_arc_k8s/day2/aks/aks_gitops_helm/_index.md

```diff
@@ -91,125 +101,151 @@ With Namespace-level GitOps config, the goal is to have Kubernetes resources dep
+- Download [the script](https://github.com/microsoft/azure_arc/blob/main/azure_arc_k8s_jumpstart/aks/gitops/helm/az_k8sconfig_helm_aks.sh) using below command.
 
-    > **NOTE: The extra dot is due to the script having an _export_ function and needs to have the vars exported in the same shell session as the rest of the commands.**
+    ```shell
+    curl -L https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_k8s_jumpstart/aks/gitops/helm/az_k8sconfig_helm_aks.sh -o ~/az_k8sconfig_helm_aks.sh
+    ```
 
-    ![Open Azure Cloud Shell](./03.png)
+    ![Open Azure Cloud Shell](./08.png)
 
-    ![Upload a file to Azure Cloud Shell](./04.png)
+    ![Download file to Azure Cloud Shell](./09.png)
 
-    ![Upload a file to Azure Cloud Shell](./05.png)
+- Edit the environment variables in the [*az_k8sconfig_helm_aks*](https://github.com/microsoft/azure_arc/blob/main/azure_arc_k8s_jumpstart/aks/gitops/helm/az_k8sconfig_helm_aks.sh) shell script to match your parameters and run it using the _`. ./az_k8sconfig_helm_aks.sh`_ command.
 
-    ![Upload a file to Azure Cloud Shell](./06.png)
+    ![Screenshot parameter](./10.png)
+
+    For example:
+
+    ![Screenshot parameter examples](./11.png)
+
+    > **NOTE: The extra dot is due to the script having an _export_ function and needs to have the vars exported in the same shell session as the rest of the commands.**
 
     The script will:
 
-  * Login to your Azure subscription using the SPN credentials
-  * Retrieve the cluster credentials (KUBECONFIG)
-  * Create two GitOps configurations for the Azure Arc Connected Cluster. Both configurations will be using the Helm charts located in the "Hello Arc" repository.
-  * Cluster-level config to deploy nginx-ingress controller Helm chart
-  * Namespace-level config to deploy the "Hello Arc" application Helm chart
+  - Login to your Azure subscription using the SPN credentials
+  - Retrieve the cluster credentials (KUBECONFIG)
+  - Create the GitOps configuration to deploy the Flux controllers and NGINX ingress controller on the Azure Arc-connected cluster
+  - Create the GitOps configuration to deploy the ["Hello Arc"](https://github.com/microsoft/azure-arc-jumpstart-apps/blob/main/hello-arc) application alongside an Ingress rule to make it available from outside the cluster
 
-    > **Disclaimer: For the purpose of this guide, notice how the "*git-poll-interval 3s*" is set. The 3 seconds interval is useful for demo purposes since it will make the git-poll interval to rapidly track changes on the repository but it is recommended to have longer interval in your production environment (default value is 5min)**
+    > **Disclaimer: For the purpose of this guide, notice how the "_sync-interval 3s_" is set. The 3 seconds interval is useful for demo purposes since it will make the sync interval rapidly track changes on the repository but it is recommended to have a longer interval in your production environment (default value is 5min)**
```

Comment:
```suggestion
    > **Disclaimer: For the purpose of this guide, notice how the "_sync-interval 3s_" is set. The 3 seconds interval is useful for demo purposes since it will make the sync interval rapidly track changes on the repository but it is recommended to have a longer interval in your production environment (the default value is 5min)**
```